Before/Office-Add-in-ASPNET-SSO-WebAPI/Scripts/Office/office_strings.debug.js

```diff
@@ -175,7 +175,7 @@ Strings.OfficeOM.L_GetSelectionNotSupported='The current selection is not suppor
 Strings.OfficeOM.L_EventHandlerAdditionFailed='Failed to add the event handler.';
 Strings.OfficeOM.L_DialogNavigateError='Dialog Navigation Error';
 Strings.OfficeOM.L_InvalidResourceUrl='Invalid application resource Url provided.';
-Strings.OfficeOM.L_InvalidOrTimedOutSessionMessage='Your Office Online session has expired or is invalid. To continue, refresh the page.';
+Strings.OfficeOM.L_InvalidOrTimedOutSessionMessage='Your Office browser session has expired or is invalid. To continue, refresh the page.';
```

Comment:
Per our discussion. Please apply to other instances as well.
```suggestion
Strings.OfficeOM.L_InvalidOrTimedOutSessionMessage='Your Office session has expired or is invalid. To continue, refresh the page.';
```